cmd/kafka-consumer/main.go

```diff
@@ -57,17 +57,25 @@ import (
 	"go.uber.org/zap"
 )
 
-// Sarama configuration options
-var (
-	kafkaAddrs           []string
-	kafkaTopic           string
-	kafkaPartitionNum    int32
-	kafkaGroupID         = fmt.Sprintf("ticdc_kafka_consumer_%s", uuid.New().String())
-	kafkaVersion         = "2.4.0"
-	kafkaMaxMessageBytes = math.MaxInt64
-	kafkaMaxBatchSize    = math.MaxInt64
-
-	downstreamURIStr string
+func newConsumerOption() *consumerOption {
+	return &consumerOption{
+		groupID: fmt.Sprintf("ticdc_kafka_consumer_%s", uuid.New().String()),
+		version: "2.4.0",
+
+		maxMessageBytes: math.MaxInt64,
+		maxBatchSize:    math.MaxInt64,
+	}
+}
+
+type consumerOption struct {
+	address      []string
+	version      string
+	topic        string
+	partitionNum int32
+	groupID      string
```

Comment:
Is `groupID ` uniquely identifying a consumer group?

Reply:
yes, it's used to identify the consumer group, but it's generated by the uuid, which means if the consumer restart, a new consumer group is added to consume message from the beginning.